src/cow-react/modules/limitOrders/state/limitOrdersPaginationAtom.ts

```diff
@@ -0,0 +1,3 @@
+import { atom } from 'jotai'
+
+export const limitOrdersPaginationAtom = atom<{ pageNumber: number }>({ pageNumber: 1 })
```

Comment:
Any reason not to store only an integer instead of an object?

Reply:
Actually no reasons

Reply:
Then can you use a number instead? :)

Reply:
Done :)